libs/ledger-live-common/src/platform/providers/RampCatalogProvider/useRampCatalog.ts

```diff
@@ -0,0 +1,47 @@
+import { useCallback, useEffect } from "react";
+import { getCryptoCurrencyIds, isCurrencyInCatalog } from "./helpers";
+import { useRampCatalogContext } from "./index";
+import type { CryptoCurrency } from "@ledgerhq/wallet-api-core/lib/currencies/types";
+
+type UseRampCatalog = {
+  isCurrencyAvailable: (
+    currencyId: CryptoCurrency["id"] | string,
+    mode: "onRamp" | "offRamp",
+  ) => boolean | null;
+  getSupportedCryptoCurrencyIds: (mode: "onRamp" | "offRamp") => CryptoCurrency["id"][] | null;
+};
+
+export function useRampCatalog(): UseRampCatalog {
+  const state = useRampCatalogContext();
+
+  useEffect(() => {}, [state.value]);
```

Comment:
This doesn't do anything?

Reply:
aha yes a hangover from a console log, thanks for spotting!!